packages/donor/src/screens/UpcommingDonation/TimeFrameAfterDonationUtil.test.ts

```diff
@@ -0,0 +1,62 @@
+import {
+  AppointmentStatus,
+  BookedAppointment,
+  Hospital,
+} from "../../../../common/src";
+import { TimeFrameAfterDonationPopup } from "./TimeFrameAfterDonationUtil";
+
+describe("Upcoming donation popup", function () {
+  const tempDate = 1658944904; // A date to compare with
+  const oneDayInMillis = 24 * 60 * 60 * 1000; // 1 day (24 hours) in milliseconds
+  const threeDaysInMillis = 259200000; // 3 days (72 hours) in milliseconds
+  const DAYS_LIMIT = 2; // 2 => 2 days => 48 hours
+  const bookedAppointment = {
+    id: "id",
+    donationStartTimeMillis: tempDate,
+    bookingTimeMillis: tempDate,
+    donorId: "donorId",
+    status: AppointmentStatus.BOOKED,
+    hospital: Hospital.TEL_HASHOMER,
+    booked: true,
+  } as BookedAppointment;
+
+  it("Should not show popup when appointment is more than 2 days (48 hours)", () => {
```

Comment:
Should not show popup when appointment is booked for more than 2 days (48 hours) in the future